pkgs/development/tools/renderizer/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "renderizer";
+  version = "2.0.5";
+
+  src = fetchFromGitHub {
+    owner = "gomatic";
+    repo = pname;
+    rev = version;
+    sha256 = "186wcfzw60z6i59yl37rkppw8w88z5kikvsi65k4r9kwpll2z6z4";
+  };
+
+  modSha256 = "1sxg9skd5hjpg2f4wyxh5hwjrplw3b3v32gn61a9yixfk3wvi05x";
+
+  meta = with stdenv.lib; {
+    description = "CLI to render Go template text files";
+    inherit (src.meta) homepage;
+    license = licenses.gpl3;
+    maintainers = with maintainers; [ yurrriq ];
+    platforms = platforms.unix;
```

Comment:
```suggestion
```

Renderer is a pure Go, so it should work everywhere. The platforms default to [Go's platform](https://github.com/NixOS/nixpkgs/blob/4d7faaa8e9350888c5707a4b2534ca1cb5917701/pkgs/development/go-modules/generic/default.nix#L217).